server/boot/root.js

```diff
@@ -3,9 +3,41 @@
 // US Government Users Restricted Rights - Use, duplication or disclosure
 // restricted by GSA ADP Schedule Contract with IBM Corp.
 
+var fs = require('fs');
+
 module.exports = function(server) {
   // Install a `/` route that returns server status
   var router = server.loopback.Router();
   router.get('/', server.loopback.status());
   server.use(router);
+
+  server.models.Project.beforeRemote('find', function(ctx, unused, next) {
```

Comment:
This only fixes the issue from the API. Might be enough to fix the bug in the designer / CLI, but kind of an odd design decision IMO.